internal/cmd/gocdk/serve.go

```diff
@@ -16,10 +16,213 @@ package main
 
 import (
 	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
 
 	"golang.org/x/xerrors"
 )
 
 func serve(ctx context.Context, pctx *processContext, args []string) error {
```

Comment:
Can you document the args? It's not obvious what `ctx` vs `pctx` is here.

Reply:
I see, never mind. I was confused by the name "Context" and for some reason thought it was another context.Context.